azure/durable_functions/decorators/metadata.py

```diff
@@ -0,0 +1,164 @@
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License.
+from typing import Optional
+
+from azure.durable_functions.constants import ORCHESTRATION_TRIGGER, \
+    ACTIVITY_TRIGGER, ORCHESTRATION_CLIENT, ENTITY_TRIGGER, ENTITY_CLIENT, \
+    DURABLE_CLIENT
+from azure.functions.decorators.core import Trigger, InputBinding
+
+
+class OrchestrationTrigger(Trigger):
```

Comment:
This is just metadata that PyStein requires to register different Function types